source/tutorials/amazon_provider_ec2.html.erb

```diff
@@ -480,12 +487,12 @@ verifier:
     - name: default
       controls:
         - operating_system
-      hostnames: public_dns
-      username: ubuntu
+      backend: ssh
```

Comment:
I see this content has fallen out of sync with the initially declared content when the user is instructed to configure the verifier. Can we make sure it matches:

```yml
verifier:
  name: terraform
  systems:
    - name: default
      controls:
        - operating_system
      backend: ssh
      user: ubuntu
      key_files:
        - ~/path/to/your/private/aws/key.pem
      hosts_output: public_dns
      reporter:
        - documentation
```